casr/src/bin/casr-libfuzzer.rs

```diff
@@ -170,9 +191,28 @@ fn main() -> Result<()> {
             casr_cmd.args(argv.clone());
             casr_cmd.arg(crash);
             debug!("{:?}", casr_cmd);
-            let casr_output = casr_cmd
-                .output()
-                .with_context(|| format!("Couldn't launch {casr_cmd:?}"))?;
+
+            // Get output
+            let casr_output =
+            // If timeout is specified, spawn and check timeout
+            // Else get output
+            if timeout != 0 {
```

Comment:
We may move this code to utility function

Reply:
Done